docs/web/tfel-material.md

```diff
@@ -15,7 +15,12 @@
 
 # Equivalent stress
 
-## Hosford equivalent stress{#sec:hosford}
+## Hosford equivalent stress{#sec:hosford}for an ellipsoid whose semi-axis lengths are a,b,c
+   * embedded in an isotropic matrix (young,nu). 
```

Comment:
Is it a copy-paste from the `C++` comments ? The `*` shall be removed.

You can generate the website with `make website` to see the results of your edits